packages/jokul/src/components/expander/styles/expandable.scss

```diff
@@ -2,6 +2,9 @@
 
 .jkl-expandable {
     background-color: var(--jkl-color-background-container-low);
+    // A transparent border will be visible in forced colors mode and replace
+    // the role of the background-color in separating the component from other content
+    border: 1px solid transparent;
```

Comment:
Denne endringen øker høyden med to piksler, og skyver innholdet mot høyre med én piksel. Vet ikke hvor mye det har å si, men det er i hvert fall grønnen til at testene knekker 😅

Reply:
Aaha, her tenkte jeg at jeg skulle briefe med at jeg vet hvordan transparente farger fungerer i forced colors 😅 

Kan bare løse det med en media-query i stedet da.